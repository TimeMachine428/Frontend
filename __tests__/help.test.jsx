import React from 'react';
import Help from '../src/js/pages/Help.jsx';
import renderer from 'react-test-renderer';

test('Help page information correct',() =>{
    const component = renderer.create(
        <Help />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})

