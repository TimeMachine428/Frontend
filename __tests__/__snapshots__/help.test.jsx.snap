// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateAccount page information correct 1`] = `
<div>
  <h1>
    Help
  </h1>
  <p>
     Welcome to Time Machine, where learning code is simple and fun.
    <br />
    Get started by logging in, then select a problem to try and solve.
    <br />
    Make sure to check the difficulty and to visit our tutorial page 
    <br />
    for extra help. For more questions, communicate with the System 
    <br />
    Administrator by clicking 
    <a
      href="mailto:admin@timemachine.com?subject=I'm lost halp.
 &body=Hi, I think your website is bad. Here's why: [insert complaint here]."
    >
      here
    </a>
    .
  </p>
</div>
`;

exports[`Help page information correct 1`] = `
<div>
  <h1>
    Help
  </h1>
  <p>
     Welcome to Time Machine, where learning code is simple and fun.
    <br />
    Get started by logging in, then select a problem to try and solve.
    <br />
    Make sure to check the difficulty and to visit our tutorial page 
    <br />
    for extra help. For more questions, communicate with the System 
    <br />
    Administrator by clicking 
    <a
      href="mailto:admin@timemachine.com?subject=I'm lost halp.
 &body=Hi, I think your website is bad. Here's why: [insert complaint here]."
    >
      here
    </a>
    .
  </p>
</div>
`;
